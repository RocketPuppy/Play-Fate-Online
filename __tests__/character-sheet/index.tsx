import * as React from "react";
import * as renderer from "react-test-renderer";
import Index from "../../src/character-sheet/index";

test("renders all the pieces", () => {
  const render = renderer.create(<Index />);

  expect(render.toJSON()).toMatchSnapshot();
});

