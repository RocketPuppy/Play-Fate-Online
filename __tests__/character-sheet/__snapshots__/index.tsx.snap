// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders all the pieces 1`] = `
<div>
  <h1>
    Character Sheet
  </h1>
  <div>
    <h2>
      Id
    </h2>
    <div>
      <label>
        <span>
          Name
        </span>
        <input
          onChange={[Function]}
          placeholder="Landon"
          type="text"
        />
      </label>
      <label>
        <span>
          Description
        </span>
        <textarea
          onChange={[Function]}
          placeholder="A rough-mannered swordsman covered in ragged clothing."
        />
      </label>
      <label>
        <span>
          Refresh
        </span>
        <input
          onChange={[Function]}
          placeholder="3"
          type="number"
        />
      </label>
    </div>
  </div>
  <div>
    <h2>
      Aspects
    </h2>
    <div>
      <label>
        <span>
          High Concept
        </span>
        <input
          onChange={[Function]}
          placeholder="Disciple of the Ivory Shroud"
          type="text"
          value=""
        />
      </label>
      <label>
        <span>
          Trouble
        </span>
        <input
          onChange={[Function]}
          placeholder="The manners of a goat"
          type="text"
          value=""
        />
      </label>
      <label>
        <span>
          New Aspect
        </span>
        <input
          onChange={[Function]}
          placeholder="I owe Old Finn everything"
          type="text"
          value=""
        />
      </label>
      <button
        disabled={true}
        onClick={[Function]}
      >
        Save Aspect
      </button>
    </div>
  </div>
  <div>
    <h2>
      Skills
    </h2>
    <form>
      <label>
        <span>
          Superb
        </span>
        <span>
          (
          <span>
            +
          </span>
          5
          )
        </span>
        <form>
          <input
            defaultValue=""
            onChange={[Function]}
            placeholder="Athletics"
            type="text"
          />
          <button
            disabled={true}
            onClick={[Function]}
          >
            Add Skill
          </button>
        </form>
      </label>
      <label>
        <span>
          Great
        </span>
        <span>
          (
          <span>
            +
          </span>
          4
          )
        </span>
        <form>
          <input
            defaultValue=""
            onChange={[Function]}
            placeholder="Athletics"
            type="text"
          />
          <button
            disabled={true}
            onClick={[Function]}
          >
            Add Skill
          </button>
        </form>
      </label>
      <label>
        <span>
          Good
        </span>
        <span>
          (
          <span>
            +
          </span>
          3
          )
        </span>
        <form>
          <input
            defaultValue=""
            onChange={[Function]}
            placeholder="Athletics"
            type="text"
          />
          <button
            disabled={true}
            onClick={[Function]}
          >
            Add Skill
          </button>
        </form>
      </label>
      <label>
        <span>
          Fair
        </span>
        <span>
          (
          <span>
            +
          </span>
          2
          )
        </span>
        <form>
          <input
            defaultValue=""
            onChange={[Function]}
            placeholder="Athletics"
            type="text"
          />
          <button
            disabled={true}
            onClick={[Function]}
          >
            Add Skill
          </button>
        </form>
      </label>
      <label>
        <span>
          Average
        </span>
        <span>
          (
          <span>
            +
          </span>
          1
          )
        </span>
        <form>
          <input
            defaultValue=""
            onChange={[Function]}
            placeholder="Athletics"
            type="text"
          />
          <button
            disabled={true}
            onClick={[Function]}
          >
            Add Skill
          </button>
        </form>
      </label>
    </form>
  </div>
  <div>
    <h2>
      Stunts
    </h2>
    <form>
      <div>
        <div>
          <label>
            <span>
              Name
            </span>
            <input
              defaultValue=""
              onChange={[Function]}
              placeholder="Lucky"
              type="text"
            />
          </label>
          <label>
            <span>
              Description
            </span>
            <textarea
              onChange={[Function]}
              placeholder="Once per session you may roll a single dice to resolve a test. If you don't fail, succeed with style automatically."
              value=""
            />
          </label>
          <button
            disabled={true}
            onClick={[Function]}
          >
            Add new stunt
          </button>
        </div>
      </div>
    </form>
  </div>
  <div>
    <h2>
      Extras
    </h2>
    <form>
      <div>
        <div>
          <label>
            <span>
              Name
            </span>
            <input
              defaultValue=""
              onChange={[Function]}
              placeholder="Izanagi"
              type="text"
            />
          </label>
          <label>
            <span>
              Description
            </span>
            <textarea
              onChange={[Function]}
              placeholder="Izanagi is available to help out during conflicts"
              value=""
            />
          </label>
          <button
            disabled={true}
            onClick={[Function]}
          >
            Add new extra
          </button>
        </div>
      </div>
    </form>
  </div>
  <div>
    <h2>
      Physical Stress (Physique)
    </h2>
    <div>
      <label>
        <span>
          1
        </span>
        <input
          checked={false}
          onChange={[Function]}
          type="checkbox"
        />
      </label>
      <label>
        <span>
          2
        </span>
        <input
          checked={false}
          onChange={[Function]}
          type="checkbox"
        />
      </label>
      <button
        onClick={[Function]}
      >
        Add Stress
      </button>
    </div>
  </div>
  <div>
    <h2>
      Mental Stress (Will)
    </h2>
    <div>
      <label>
        <span>
          1
        </span>
        <input
          checked={false}
          onChange={[Function]}
          type="checkbox"
        />
      </label>
      <label>
        <span>
          2
        </span>
        <input
          checked={false}
          onChange={[Function]}
          type="checkbox"
        />
      </label>
      <button
        onClick={[Function]}
      >
        Add Stress
      </button>
    </div>
  </div>
  <div>
    <h2>
      Consequences
    </h2>
    <div>
      <h3>
        Normal
      </h3>
      <div>
        <label>
          <span>
            2
          </span>
          <span>
            Mild
          </span>
          <input
            defaultValue=""
            onChange={[Function]}
            type="text"
          />
        </label>
        <label>
          <span>
            4
          </span>
          <span>
            Moderate
          </span>
          <input
            defaultValue=""
            onChange={[Function]}
            type="text"
          />
        </label>
        <label>
          <span>
            6
          </span>
          <span>
            Severe
          </span>
          <input
            defaultValue=""
            onChange={[Function]}
            type="text"
          />
        </label>
      </div>
      <h3>
        Physical
      </h3>
      <div />
      <button
        onClick={[Function]}
      >
        Add Consequence
      </button>
      <h3>
        Mental
      </h3>
      <div />
      <button
        onClick={[Function]}
      >
        Add Consequence
      </button>
    </div>
  </div>
</div>
`;
